src/App.css

```diff
@@ -4,8 +4,15 @@
     box-sizing: border-box;
 }
 
-.parent-container {
+/* Only for sample containers of Home, Past Events, Join Us and Contact Us Components */
+
+.containers {
```

Comment:
we no longer need containers syle here right?

as we are not using it in App.jsx so you delete this .containers{} and 

```
@media only screen and (max-width: 500px) {
    .containers {
        padding-top: 85px; /* For better view in mobile screens */
    }
}
```

Reply:
I haven't created separate css files, for the Home, Past Events components, etc and so have imported App.css in all, and given styles to the containers present in it. So, we working on any of the component, one can remove at that time, for now I have kept it to demonstrate the working of the navbar

Reply:
imo we can remove these components for now:
```
<Home/>
<PastEvents/>
<JoinUs/>
<ContactUs/>
```

They'll be made/worked upon in the upcoming issue

Reply:
Ok. So, after removing the the components and the style from the App.css and the chkraui dependency, should I create another pr?

Reply:
> Ok. So, after removing the the components and the style from the App.css and the chkraui dependency, should I create another pr?

No you just need to repeat the same git add, git commit and git push steps again (it will be reflected here once you push again)